manifests/instance/config.pp

```diff
@@ -163,6 +165,19 @@
     }
   }
 
+  # configure JMX
+
+  $jmxremote_defaults = {
+    'port'         => '9010',
+    'local.only'   => true,
+    'authenticate' => false,
+    'ssl'          => false,
+  }
```

Comment:
Yes. It's up to the user to deal with it when he has several tomcat instances on the same node.